.github/workflows/deploy.yml

```diff
@@ -21,10 +21,18 @@ jobs:
       - name: Build
         run: CI=false npm run build
 
-      - name: Deploy to Skynet
+      - name: Deploy Application to Skynet
         uses: SkynetLabs/deploy-to-skynet-action@v2
         with:
           upload-dir: build
           github-token: ${{ secrets.GITHUB_TOKEN }}
           registry-seed: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' && secrets.REGISTRY_SEED || '' }}
           registry-datakey: ${{ secrets.REGISTRY_DATAKEY || '' }}
+
+      - name: Deploy Featured to Skynet
+        uses: SkynetLabs/deploy-to-skynet-action@v2
+        with:
+          upload-dir: featured
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          registry-seed: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' && secrets.REGISTRY_SEED || '' }}
+          registry-datakey: "featured"
```

Comment:
I would make this a github secret so that we don't break it by someone changing it in a PR.

Let me know what you would like it to be called and I can add it.